beacon_node/lighthouse_network/src/types/topics.rs

```diff
@@ -23,21 +23,47 @@ pub const BLS_TO_EXECUTION_CHANGE_TOPIC: &str = "bls_to_execution_change";
 pub const LIGHT_CLIENT_FINALITY_UPDATE: &str = "light_client_finality_update";
 pub const LIGHT_CLIENT_OPTIMISTIC_UPDATE: &str = "light_client_optimistic_update";
 
-pub const CORE_TOPICS: [GossipKind; 7] = [
+pub const BASE_CORE_TOPICS: [GossipKind; 5] = [
     GossipKind::BeaconBlock,
     GossipKind::BeaconAggregateAndProof,
     GossipKind::VoluntaryExit,
     GossipKind::ProposerSlashing,
     GossipKind::AttesterSlashing,
-    GossipKind::SignedContributionAndProof,
-    GossipKind::BlsToExecutionChange,
 ];
 
+pub const ALTAIR_CORE_TOPICS: [GossipKind; 1] = [GossipKind::SignedContributionAndProof];
+
+pub const CAPELLA_CORE_TOPICS: [GossipKind; 1] = [GossipKind::BlsToExecutionChange];
+
+pub const EIP4844_CORE_TOPICS: [GossipKind; 1] = [GossipKind::BeaconBlocksAndBlobsSidecar];
+
 pub const LIGHT_CLIENT_GOSSIP_TOPICS: [GossipKind; 2] = [
     GossipKind::LightClientFinalityUpdate,
     GossipKind::LightClientOptimisticUpdate,
 ];
 
+/// Returns the core topics associated with each fork
```

Comment:
```suggestion
/// Returns the core topics associated with each fork that are new to the previous fork
```
Just a suggestion to make clear this is not the complete set of topics for the fork, but the new ones to add